src/components/AsideHeader/components/FooterItem/FooterItem.module.scss

```
@@ -4,4 +4,6 @@ $block: '.#{variables.$ns}footer-item';
 
 #{$block} {
     width: 100%;
+
+    border-radius: var(--gn-aside-header-item-expanded-radius);
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)